asistente_ladm_col/lib/dbconnector/pg_connector.py

```diff
@@ -47,6 +47,20 @@ def _schema_exists(self):
 
         return False
 
+    def _postgis_exists(self):
+        if self.schema:
+            cur = self.conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+            cur.execute("""
+                        SELECT 
+                            count(extversion)
+                        FROM pg_catalog.pg_extension
+                        WHERE extname='postgis'
+            """.format(self.schema, INTERLIS_TEST_METADATA_TABLE_PG))
```

Comment:
You don't need to call the `format` function nor pass any parameter to such SQL query, do you?.

Reply:
Fixed in https://github.com/AgenciaImplementacion/Asistente-LADM_COL/commit/11bcc5ceee4ae56ed1a60855067fcd9d9c7a8589